README.md

```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://craftwork.design/downloads/draw/"><img width='700' src='https://user-images.githubusercontent.com/74038190/195007270-35b11987-f822-46f9-840a-6e8db7e91719.png' alt="DrawFree_free_illustrations" /></a> | [Draw Free](https://craftwork.design/downloads/draw/)  <br><br> ✔ Free  <br><br>Format <br> • Ai<br> • PNG<br> • SVG<br><br> ✌️Author <br> Craftwork |
 | <a href="https://doodleipsum.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643856-ea39f8f7-6c5d-45a5-a474-65f38063594b.png' alt="Doodle_Ipsum_free_illustrations" /></a> | [Doodle Ipsum](https://doodleipsum.com/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Blush |
 | <a href="https://drawer.design/products/avatar-library/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184863865-4c6200a4-cd20-4695-a330-9ae0f9735b55.png' alt="Drawer_Avatar_free_illustrations" /></a> | [Drawer Avatar](https://drawer.design/products/avatar-library/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br><br> ✌️Author <br> Drawer                                                                  |
-| <a href="https://drawkit.com/product/ecology-environment-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479444-40aa7a75-b8df-4736-a4f0-8001cd362f5b.png' alt="ecology_&_environment_free_illustrations" /></a> | [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit |
+| <a href="https://drawkit.com/product/ecology-environment-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479444-40aa7a75-b8df-4736-a4f0-8001cd362f5b.png' alt="ecology_&_environment_free_illustrations" /></a> | [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit   |
+| <a href="https://drawkit.com/product/ecology-nature-free-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194713342-0a5264a2-1ebe-4497-9dee-f7136f29033e.png' alt="ecology_&_nature_free_illustrations" /></a> | [Ecology & Nature](https://drawkit.com/product/ecology-nature-free-icons)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
+| <a href="https://www.drawkit.com/product/free-education-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194964105-d0659d45-1c5f-4a31-8f29-2c82333232fe.png' alt="education_icons_free_illustrations" /></a> | [Education Icons](https://www.drawkit.com/product/free-education-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit   |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195569147-2bcdf4bc-5fd4-4823-a306-6ef08039cc6d.png` in Education Icons